### PR TITLE
Enable generic Grafana dashboards

### PR DESCRIPTION
**Pull request by skabashnyuk**
- Generic operator SDK dashboard
 - Generic go process dashboard

### CODE REVIEW

File: components/monitoring/grafana/base/kustomization.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ resources:
 - release/
 - performance/
 - pipeline-service/
+- dashboards/
```
Comment (amisstea) [inline components/monitoring/grafana/base/kustomization.yaml:18]: Can we rename this directory? `dashboards` is too generic IMO. All other directories referenced here contain dashboards.

**Review by amisstea — COMMENTED** [review]
I'd like to get @avi-biton's input on this. The new guidelines are to create dashboards in team specific repos but that might be too restrictive when dealing with generic dashboards like these.

File: components/monitoring/grafana/base/kustomization.yaml (hunk 1 above)
Comment (avi-biton) [reply components/monitoring/grafana/base/kustomization.yaml:18]: I think that `generic-dashboards` would be a good name for the directory

**Comment by avi-biton** [comment]
@amisstea although the guidelines are to create dashboards in team specific repos, but as you mentioned, this is a generic dashboard and not a team's dashboard, I believe it can be created the way it is (in `infra-deployments`).

**Review by avi-biton — COMMENTED** [review]
/lgtm

**Review by amisstea — APPROVED** [review]
/lgtm

**Pull request merged**